packages/react-native/.npmignore

```diff
@@ -0,0 +1,80 @@
+# OSX
+#
+.DS_Store
+
+# Xcode
+#
+build/
+*.pbxuser
+!default.pbxuser
+*.mode1v3
+!default.mode1v3
+*.mode2v3
+!default.mode2v3
+*.perspectivev3
+!default.perspectivev3
+xcuserdata
+*.xccheckout
+*.moved-aside
+DerivedData
+*.hmap
+*.ipa
+*.xcuserstate
+project.xcworkspace
+
+# Android/IntelliJ
+#
+build/
+.idea
+.gradle
+local.properties
+*.iml
+
+# node.js
+#
+node_modules/
+npm-debug.log
+yarn-error.log
+
+# BUCK
+buck-out/
+\.buckd/
+*.keystore
+
+# fastlane
+#
+# It is recommended to not store the screenshots in the git repo. Instead, use fastlane to re-generate the
+# screenshots whenever they are needed.
+# For more information about the recommended setup visit:
+# https://docs.fastlane.tools/best-practices/source-control/
+
+*/fastlane/report.xml
+*/fastlane/Preview.html
+*/fastlane/screenshots
+
+# Bundle artifact
+*.jsbundle
+
+# Haul
+#
+haul-debug.log
+
+# Ignore generated `storybook/stories/index.js`
+storybook/stories/index.js
+metro.config.js
+
+# The files are added using the `react-native eject` command for development
+ios
+android
+
+### ▲ .gitignore contents above
+### ▼ additional ignore files for publishing
+
+# Coverage directory used by tools like istanbul
+coverage
+.nyc_output
```

Comment:
shouldn't those 2 be in `.gitignore` also?

Reply:
They are in the top-level `.gitignore` for the entire repository.

Reply:
Also addressed in https://github.com/godaddy/exemplar/pull/11#discussion_r311793282